src/LookupTable/components/list/ListTable.tsx

```diff
@@ -2,30 +2,20 @@ import React from 'react'
 import { NavLink } from 'react-router-dom'
 import { Button, Header, Icon, Message, Popup, Table } from 'semantic-ui-react'
 import { Loading } from '../../../components'
-import { FieldMapType, LookUpTableType } from '../../types'
+import { LookUpTableType } from '../../types'
 import { DownloadButton } from '..'
 import { useRouter } from '../../../utils/hooks/useRouter'
 import { useLanguageProvider } from '../../../contexts/Localisation'
 
 const TABLE_PREFIX = 'data_table_'
 
-const ListTable: React.FC<any> = ({
-  allTableStructures,
-  allTableStructuresLoadState,
-  setAllTableStructures,
-}: any) => {
+const ListTable: React.FC<any> = ({ allTableStructures, allTableStructuresLoadState }: any) => {
   const { t } = useLanguageProvider()
   const { loading, error } = allTableStructuresLoadState
   const {
     match: { path },
   } = useRouter()
 
```

Comment:
Expansion not used anymore ?

Reply:
Nope, see earlier comment: https://github.com/msupply-foundation/conforma-web-app/pull/1639#discussion_r1637246584